packages/ui/src/lib/Navbar/index.tsx

```diff
@@ -0,0 +1,99 @@
+import { Image, Box, HamburgerIcon, HStack, IconButton } from "native-base";
+import PropTypes, { InferProps } from "prop-types";
+import React, { useReducer } from "react";
+import MenuItem, { menuItemPropTypes } from "./MenuItem";
+
+const navbarPropTypes = {
+  items: PropTypes.arrayOf(PropTypes.exact(menuItemPropTypes)).isRequired,
+};
+
+type NavbarProps = InferProps<typeof navbarPropTypes>;
+
+const style = {
+  navbar: {
+    height: {
+      base: "50px",
+      md: "70px",
+    },
+    px: {
+      base: "15px",
+      md: "30px",
+    },
+  },
+  logo: {
+    width: {
+      base: "101px",
+      md: "144px",
+    },
+    height: {
+      base: "35px",
+      md: "50px",
+    },
+  },
+  flexOnSmallScreen: {
+    display: {
+      base: "flex",
+      md: "none",
+    },
+  },
+  flexOnBigScreen: {
+    display: {
+      base: "none",
+      md: "flex",
+    },
+  },
+};
+
+export default function Navbar({ items }: NavbarProps) {
+  const [isMenuOpen, toggleMenu] = useReducer(
+    (isOpen: boolean) => !isOpen,
```

Comment:
Shall we move `(isOpen: boolean) => !isOpen` out of renders?

Reply:
Yes, I should do it.